index.html

```diff
@@ -0,0 +1,190 @@
+<!DOCTYPE html>
+<html lang="en">
+  <!--HEAD-->
+  <head>
+    <meta charset="UTF-8" />
+    <title>Tech Doc | Nisar & Besu</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+
+  <!--BODY-->
+  <body>
+    <!--NAVIGATION BAR-->
+    <div id="navbar">
+      <header>JavaScript Documentation</header>
+      <ul>
+        <li><a class="nav-link" href="#Introduction">Introduction</a></li>
+        <li>
+          <a class="nav-link" href="#What_you_should_already_know"
+            >What you should already know
+          </a>
+        </li>
+        <li><a class="nav-link" href="#Variables">Variables</a></li>
+        <li><a class="nav-link" href="#Variable_scope">Variable scope</a></li>
+        <li><a class="nav-link" href="#Constants">Constants</a></li>
+        <li><a class="nav-link" href="#Reference">Reference</a></li>
+      </ul>
+    </div>
```

Comment:
I think it would be better if you put this section into the nav tag which will be more semantic.